### DISCUSSION TITLE
How to enable https in apisix ingress gateway?

### DISCUSSION THREAD

**Discussion by MirtoBusico** [apache/apisix#6244] [Q&A] [answered]
Hi all,
I've defined a route and an upstream for "http://www.m01.net" and it works correctly
Now I want to enable https so that the route replyes to "https://www.m01.net"

My framework:

- 3 vm kubernetes nodes with k3s
- Istio installed with "minimal" profile
- apisix, ingress controller and dashboard installed via helm chart
- apisix gateway exposed as LoadBalancer
- www.m01.net is resolved to the first k3s node address

 Found some documentation but I was not able to understand what to do.

Any example and/or documentation?

**Comment by tzssangglass** [comment, marked as answer]
PLAT: https://github.com/apache/apisix/blob/master/docs/en/latest/certificate.md

**Reply by MirtoBusico** [reply]
For me it is working using apisix dashboard. Are the two methos equivalent?
<img width="1590" alt="ksnip_20220209-092734" src="https://user-images.githubusercontent.com/11090934/153155832-ac63cdff-c177-4182-b65c-b4e2ecfde63d.png">

**Reply by tokers** [reply]
For Ingress Controller, creating `ApisixTls` resources (https://apisix.apache.org/docs/ingress-controller/references/apisix_tls), it's equivalent to create SSL resource here.

**Reply by MirtoBusico** [reply]
Thanks
